openage/launcher/mod.py

```diff
@@ -0,0 +1,64 @@
+import toml
+
+
+class ModInfo:
+    """
+    Parses and exposes info about a mod
+    """
+    def __init__(self, file_path=None, url=None):
+        if file_path is not None:
+            with open(file_path, 'r') as f:
+                info = toml.load(f)
+        elif url is not None:
+            info_str = some_black_magic(url)    # TODO: pseudocode
+            info = toml.loads(info_str)
+        else:
+            raise Exception
+ 
+        self.info = info
+
+        # for convenience
+        self.uid = self.info['uid']
+        self.version = self.info['version']
+        self.author = self.info['author']
+        self.confl = self.info['conflicts']
+        self.req = self.info['requires']
+
+    def conflicts_with(self, other):
```

Comment:
Use a function `get_conflicts()` for returning `self.conflicts` instead and solve the conflicts on a higher level in the launcher. Something like `get_conflicts()` is just easier to work with because you can also use it in other situations like displaying the conflicts or searching them for a specific mod UID.